### DISCUSSION TITLE
Can I get an invite for Concourse CI on Slack?

### DISCUSSION THREAD

**Discussion by martijnvandenboom** [concourse/concourse#8445] [Help & Support]
Dear reader,

Is it still possible to participate with Concourse in the Slack environment?

The old link is not working anymore : http://slack.concourse.ci/
Can you look into this?
Is there maybe a new one?

Can I get an invite for Concourse CI on Slack on my e-mail address?
E-mail: martijn.vandenboom@icloud.com

Sincerely,

ing. M.A.C.M. (Martijn) van den Boom

**Comment by marco-m** [comment]
See https://concourse-ci.org/project.html#where-is-everything for a link to the Discord server.
